### DISCUSSION TITLE
Trovis 5578 Anbindung

### DISCUSSION THREAD

**Discussion by HXP2** [Tom-Bom-badil/samson_trovis_557x#7] [General (de/en)]
Der Trovis 5578 Regler hat genau eine RJ45 Buchse an der Rückseite des Gerätes. Die hier im Wiki beschriebene Verbindung mittels USR-232-E2 funktioniert nicht, die Buchse scheint keine RS232 Schnittstelle zu sein. Fehlersuchergebnis: An den beschriebenen Pins für +5V und GND liegt keine Spannung an.

Im Handbuch kommt der Begriff RJ-45 genau einmal vor, mit folgendem Text:

Optionale Schnittstellen Modbus-Schnittstelle RS-485 für Zweileiterbus mit Kommunikationsmodul RS-485
(Modbus RTU-Protokoll, Datenformat 8N1, Anschlussbuchse seitlich RJ-45).

Das seitlich ist hoffentlich nur ein Fehler im Handbuch, die Buche ist wie gesagt auf der Rückseite. 

Nun ergeben sich 2 Fragen:
1) Welche Pinbelegung hat die RJ45 Buchse? In der Dokumentation und beim Kundenservice war ich nicht erfolgreich. 
2) Kann man mit einem RS485 to Ethernet TCP/IP Modbus Gateway den Regler wie in diesem Projekt beschrieben steuern bzw. auslesen?

**Comment by Tom-Bom-badil** [comment]
Bei den Pins kann ich nur raten - vermutlich 1+8.

RS485 oder 232 ist auf der seriellen Seite egal, es muss natürlich die passende Schnittstelle bzw der passende Adapter sein - hier halt 485. Siehe [hier](https://github.com/Tom-Bom-badil/samson_trovis_557x/wiki/Begriffe-und-'Basics'#protokoll---modbus-für-dummies).

Viel Erfolg!

/tom

_Edit: Gerade nochmal nachgesehen - es gibt eine bestehende 5578-Anbindung über RS232. 1 TxD, 2 RxD, 8 GND. Siehe [hier](https://github.com/Tom-Bom-badil/samson_trovis_557x/wiki/Modellspezifische-Informationen#5578)._
